src/__internal__/utils/helpers/tags/tags-specs/tags-specs.ts

```diff
@@ -22,5 +22,16 @@ const rootTagTest = (
   expect(rootNode.prop("data-role")).toEqual(role);
 };
 
+const rootTagTestRtl = (
```

Comment:
Comment: clearly great minds think alike 😄, I've actually added the same test as part of this open PR https://github.com/Sage/carbon/pull/6345/files#diff-1bd31fff78b0f0250dcf9b00bd118dee6dbf37b35d805e3258793861ad870162. I guess whichever gets merged last should just consume the one from master